### PR TITLE
Add client name to logs

### PR DESCRIPTION
**Pull request by Tzurz**

### CODE REVIEW

File: src/accounts/add/add-account-dialog-handler.js [hunk 1]
```diff
@@ -168,19 +167,25 @@ class AddAccountDialogHandler {
...
+              companyName
+            })
+          );
+
+          if (this.hasNoDefaultWorkspace(defaultConfig, alias, apiToken)) {
+            //first one or missing
+            this.teamConfigService.saveDefault(team.id, config);
+          }
+          bot.dialogOk();
+        });
```
Comment (nzyoni) [inline src/accounts/add/add-account-dialog-handler.js:188]: consider to async/await this block (not blocker at all)

File: src/accounts/channel/set-channel-account-command.js [hunk 2]
```diff
@@ -3,20 +3,29 @@ const commandRegexWithAlias = /set channel account (.*)/;
 const commandRegex = /set channel account/;
 const LoggerFactory = require('../../core/logging/logger-factory');
 const logger = LoggerFactory.getLogger(__filename);
+const { logEvent } = require('../../core/logging/logging-service');
```
Comment (nzyoni) [inline src/accounts/channel/set-channel-account-command.js:6]: remove :)

File: src/accounts/add/setup-command.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 const Command = require('../../core/commands/command');
 const LoggerFactory = require('../../core/logging/logger-factory');
-const { getEventMetadata } = require('../../core/logging/logging-metadata');
+const { logEvent } = require('../../core/logging/logging-service');
```
Comment (nzyoni) [inline src/accounts/add/setup-command.js:3]: remove :)

File: src/accounts/default/set-default-command.js [hunk 4]
```diff
@@ -2,21 +2,30 @@ const Command = require('../../core/commands/command');
 const LoggerFactory = require('../../core/logging/logger-factory');
 const commandRegexWithAlias = /set workspace account (.*)/;
 const commandRegex = /set workspace account/;
-const logger = LoggerFactory.getLogger(__filename);
+const { logEvent } = require('../../core/logging/logging-service');
```
Comment (nzyoni) [inline src/accounts/default/set-default-command.js:5]:
```suggestion
```

File: src/accounts/default/set-default-command.js [hunk 5]
```diff
@@ -25,6 +34,13 @@ class SetWorkspaceAccountCommand extends Command {
       [commandRegex],
       'direct_message,direct_mention',
       (bot, message) => {
+        this.reportCommandWithCompanyName({
+          userObject: message,
+          teamConfigurationService: this.teamConfigurationService,
+          logger,
+          eventName: 'set-default-account',
+          action: 'triggered the set default account command'
+        });
```
Comment (nzyoni) [inline src/accounts/default/set-default-command.js:43]: [Line 22](https://github.com/logzio/slack-integration/pull/109/files?file-filters%5B%5D=.js#diff-76dbe80c012232a3652541ac3041fd088db35760a51195cafd9d5ab73d0c88e0R22)
same log for different actions(are they?), is that ok?

File: src/core/configuration/team-configuration-service.js [hunk 6]
```diff
@@ -97,10 +101,24 @@ class TeamConfigurationService {
...
         if (!result) {
-          logger.error("tc:"+result+",alias" + teamConfiguration.getAlias()+",teamId="+teamId)
+          logger.error(
+            'tc:' +
+              result +
+              ',alias' +
+              teamConfiguration.getAlias() +
+              ',teamId=' +
+              teamId
+          );
```
Comment (nzyoni) [inline src/core/configuration/team-configuration-service.js:111]: Can you use string literal here?
```suggestion
  logger.error(`tc: ${result}, alias ${teamConfiguration.getAlias()} teamId= ${teamId}`);
```

**Review by nzyoni — APPROVED** [review]
Very noice
let's talk again about `getCompanyNameForTeamId` before merging it if you don't mind :)

File: src/core/commands/command.js [hunk 7]
```diff
@@ -28,6 +29,23 @@ class Command{
...
+    action,
+    eventName,
+    logger,
+    teamConfigurationService
+  }) {
+    const teamId =
+      typeof userObject.team === 'string'
+        ? userObject.team
+        : userObject.team.id;
+    const companyName = await teamConfigurationService.getCompanyNameForTeamId(
```
[the remark this answers is not in the document]
Comment (Tzurz) [reply src/core/commands/command.js:44]: no :( only saved in db

File: src/accounts/default/set-default-command.js (hunk 5 above)
Comment (Tzurz) [reply src/accounts/default/set-default-command.js:43]: yeah, cuz it's the same command but with alias (more param)

File: src/core/configuration/team-configuration-service.js (hunk 6 above)
Comment (Tzurz) [reply src/core/configuration/team-configuration-service.js:111]: I don't mind, just saying this change here is just because I did prettier.

**Comment by Tzurz** [comment]
merging so we can check in staging, will open a PR to apply PR corrections.

**Pull request merged**